Source/effects.cpp

```diff
@@ -10,6 +10,13 @@
 #include "sound.h"
 
 namespace devilution {
+namespace {
+#ifndef DISABLE_STREAMING_SOUNDS
+constexpr bool AllowStreaming = true;
+#else
+constexpr bool AllowStreaming = false;
+#endif

```

Comment:
Is there any way to simplify this? Any way to do something like:
```cpp
constexpr bool AllowStreaming = TO_BOOL(DISABLE_STREAMING_SOUNDS);
```
?

Reply:
I can't think of a way that wouldn't be "clever"